server/src/test/java/org/apache/druid/metadata/SqlSegmentsMetadataManagerTest.java

```diff
@@ -712,8 +713,12 @@ public void testMarkAsUsedNonOvershadowedSegmentsInIntervalWithInvalidInterval()
     publish(newSegment1, false);
     publish(newSegment2, false);
     // invalid interval start > end
-    final Interval theInterval = Intervals.of("2017-10-22T00:00:00.000/2017-10-02T00:00:00.000");
-    sqlSegmentsMetadataManager.markAsUsedNonOvershadowedSegmentsInInterval(newDataSource, theInterval);
+    DruidExceptionMatcher.invalidInput().assertThrowsAndMatches(
+        () -> {
+          Interval theInterval = Intervals.of("2017-10-22T00:00:00.000/2017-10-02T00:00:00.000");
+          sqlSegmentsMetadataManager.markAsUsedNonOvershadowedSegmentsInInterval(newDataSource, theInterval);
+        }
+    );
```

Comment:
Actually, when looking at this test, it seems to be throwing when you read the interval, and that has absolutely no relation to the MetadataManager or any fo the setup.  I think this test is just useless and should be deleted whole-hog.